fem/estimators.hpp

```diff
@@ -598,6 +598,8 @@ class KellyErrorEstimator final : public ErrorEstimator
    flux_space; /**< @brief Ownership based on own_flux_fes. */
    bool own_flux_fespace; ///< Ownership flag for flux_space.
 
+   bool with_coef; ///< Whether the flux or gradient should be used.
```

Comment:
This comment was a surprise, which suggests the name "with_coef" is too vague. Consider a better name, `using_flux` or some such?

Reply:
I can do this, but want to note that I chose this name to be consistent with the remaining error estimators. Should I change the naming there too?